lsdlabs/to-do-coronavirus-edition
# Liz button fix

## Merge after the following PRs ⚠️
NA

## What does this PR do? 🔧
This PR is branched off the my-watchlist-button branch (so it has all those changes) except it removes the Button view from the NavigationLink.

## How should it be manually tested? 🔎
You should see the image that was originally added to the button on the landing screen and when you tap it, it should navigate you to the detail screen.

## Any background context you want to provide, or implementation details you want to call out? 📖


## Screenshots/GIFs 📸
